yarn.lock

```diff
@@ -4788,7 +4788,7 @@ __metadata:
   languageName: node
   linkType: hard
 
-"protobufjs@npm:^6.10.0, protobufjs@npm:^6.11.2, protobufjs@npm:^6.8.8":
+"protobufjs@npm:^6.10.0, protobufjs@npm:^6.8.8":
```

Comment:
Yarn still keeps the old version, please run `yarn dedupe` to actually remove the vulnerabble version.

Reply:
Thanks.. I forgot about that!